Tools/autotest/sim_vehicle.py

```diff
@@ -877,6 +885,14 @@ def start_mavproxy(opts, stuff):
     progress("Sleeping for %f seconds" % (cmd_opts.delay_start,))
     time.sleep(float(cmd_opts.delay_start))
 
-start_mavproxy(cmd_opts, frame_infos)
+try:
+    if not cmd_opts.no_mavproxy:
+        start_mavproxy(cmd_opts, frame_infos)
+    else:
+        time.sleep(2)
+        progress("Running SITL in new terminal ... Waiting for exit")
+        wait_ulimited()
+except KeyboardInterrupt:
+    progress("Interrupt received, stopping...")
```

Comment:
Probably near enough, but keyboard interrupts are only one sort of interrupt; could be "Keyboard Interrupt received ..."